gen3/bin/config-env.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# This script will copy the environment configuration of the specified environment or apply
+# version updates to all the services and may specify particular values for a given subset of services
+
+source ${GEN3_HOME}/gen3/lib/utils.sh
+
+tgt_env=~/cdis-manifest/${USER}.planx-pla.net
```

Comment:
that's cool! i think this will only work in Dev and QA environments - which is what we want anyway -  again, please mention that in the top comment